pep-0001.txt

```diff
@@ -293,14 +294,16 @@ the time when when Python was led by `a BDFL <Python's BDFL_>`_.
 Any legacy references to "BDFL-Delegate" should be treated as equivalent to
 "PEP-Delegate".
 
-Individuals offering to serve as PEP-Delegate should notify the Steering
-Council, PEP authors and PEP sponsor of their intent to self-nominate.
+Individuals offering to serve as PEP-Delegate should notify the PEP authors
+and PEP sponsor of their intent to self-nominate, and submit their request as
+`a new issue <Steering Council issue_>`_ on the `Steering Council repository`_.
 Those taking on this responsibility are free to seek
 additional guidance from the Steering Council at any time, and are also expected
 to take the advice and perspectives of other core developers into account.
 
-Such self-nominations are accepted by default, but may be explicitly declined by
-the Steering Council. Possible reasons for the Steering Council declining a
+The Steering Council will generally accept such self-nominations by default,
+but may choose to decline them.
```

Comment:
```suggestion
The Steering Council will ordinarily approve such self-nominations,
but may choose to decline them.
```

If "ordinarily" is too formal, "usually" also works.

"approve" makes more explicit that you must hear back in the affirmative from the Council.

A

Reply:
I see what you're trying to do here, but I'm concerned it is getting a bit far away from the original intended meaning—it now implies that the SC will typically approve nominations, but no longer directly mentions the decision-making rationale of approve-by-default, the original purpose of this sentence. It is now making a statement on approval _probabilty_ (offers will be ordinarily approved) rather than approval _strategy_ (offers will be approved by default unless there is a significant reason not to do so), as "ordinarily" (and "usually") refers to frequency while "generally" more generically is a general hedge, while "accept [...] by default" was what "generally" was referencing, not "accept" alone.

A more conservative suggestion that keeps the original meaning but applies your "approve" change would be the following:

```suggestion
The Steering Council will generally approve such self-nominations by default,
but may choose to decline them.
```

Reply:
Also, @brettcannon, thoughts?

Reply:
Thanks for the added rationale -- the "generally" and "by default" felt odd in the same sentence, but I see they are serving different purposes.